CMakeLists.txt

```diff
@@ -75,18 +75,18 @@ if(UNIX OR MINGW)
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
 endif()
 
-include("cmake/mkldnn_compat.cmake")
-
-include("cmake/utils.cmake")
-include("cmake/options.cmake")
-include("cmake/OpenMP.cmake")
-include("cmake/TBB.cmake")
-include("cmake/OpenCL.cmake")
-include("cmake/platform.cmake")
-include("cmake/SDL.cmake")
-include("cmake/MKL.cmake")
-include("cmake/Doxygen.cmake")
-include("cmake/version.cmake")
+include(mkldnn_compat)
```

Comment:
Have you verified that this does not break when building DNNL as a subproject?

Reply:
We're already building DNNL as a subproject and so far we haven't seen any issues.

Reply:
The changes in CMakes seem to break building the library as a project though.

```
$ cmake --version
cmake version 2.8.11
$ cmake ..
-- CMAKE_BUILD_TYPE is unset, defaulting to Release
-- GPU support is disabled
CMake Error at cmake/utils.cmake:99 (target_link_libraries):
  Object library target "dnnl_common" may not link to anything.
Call Stack (most recent call first):
  src/common/CMakeLists.txt:27 (target_link_libraries_build)
 
 
-- Configuring incomplete, errors occurred!
```

Reply:
It builds fine on my machine as a project as well, but I'm using a much more recent version of CMake (3.13.4).

Reply:
Closing the discussion as the changes [are not relevant anymore](https://github.com/intel/mkl-dnn/pull/572#issuecomment-544695471)